Makefile

```diff
@@ -29,6 +29,10 @@ docs: docs/conf.py docs/index.rst
 tests:
 	$(BIN)python -m pytest tests/
 
+#Preprocessing pipelines
+preprocess_data:
+	(cd src/data/ ; python data_optimizer_script.py)
```

Comment:
Try `$(BIN)python -m src.data.data_optimizer_script` so we don't have `cd` potentially messing things up